Docs/README_publish.txt

```diff
@@ -0,0 +1,29 @@
+1. Zainstalować linuxa
+2. Zainstalować niezbędne narzędzia Windows do komunikacji z linuxem
+	-putty  (klient SSH)
+	-WIN SCP (protokół SCP do wysyłki plików WINDOWS <-> LINUX)
+3. Zainstalować PostgreSQL na Linuxie
+	-wyedytować wpis listen_addresses w postgresql.conf
+	-wyedytować PG_HBA.conf
+4. Utworzyć katalog na projekt i nadać mu uprawnienia (polecenie CHMOD)
+5. Zainstalować na Linuxie narzędzia:
+	net core SDK 2.2: https://dotnet.microsoft.com/download/linux-package-manager/ubuntu18-04/sdk-current
+	nginx: sudo apt-get install nginx
+6. Wyedytować plik /etc/nginx/sites-available/default
+	server {
+    		listen 80;
+  	location / {
+    		 proxy_pass http://localhost:5000;
+        	proxy_http_version 1.1;
+	        proxy_set_header Upgrade $http_upgrade;
+	        proxy_set_header Connection keep-alive;
+	        proxy_set_header Host $host;
+	        proxy_cache_bypass $http_upgrade;
+    		}
+	}
+7. Opdalić nginx: 
+	-sudo nginx-t
+	-sudo service nginx start
+8. Wykonać polecenie: dotnet publish na Windowsie
+9. Przeżucić pliki do katalogu Linuxowego za pomocą WinSCP
+10. dotnet run WebAPI.dll
```

Comment:
It would be nice to keep 2 versions of README_publishPL.txt and README_publishEN.txt  if we would like anyone else to join or just to keep it in EN version.